dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -58,6 +58,24 @@
   = f.select :video_key, options_for_select(video_key_choices, @level.video_key), {include_blank: true}, {class: 'video-dropdown'}
   .video-preview{style: 'display: block'}
 
+.field
+  = f.label :map_reference, 'Map Reference'
+  %p If there is a key resource you want to reference in the instructions, add the link here. It will show up under the 'Help & Tips' tab.
+  %p This should be a string representing the URL on docs.code.org you want to embed, i.e. '/cs/maker_leds/index.html'
```

Comment:
Your example link should be csd instead of cs, and should also be preceeded by /docs (this was changed recently), i.e. `/docs/csd/maker_leds/index.html`